src/js/BookReader.js

```diff
@@ -173,6 +176,7 @@ BookReader.prototype.setup = function(options) {
    * calls at the correct moments.
    **/
   this._components = {
+    bookModel: new BookModel(this),
```

Comment:
Yeah, this isn't the best usage of the word "component" :P ~I tried, but couldn't find an elegant way to generalize the `exposeComponentMethod` function right now, so punting on doing that after we have everything in components (when I think it'll be easier to see what the best pattern should be). I think long term `this._model` would be best. I'll add a comment for now :+1:~ We found a good way to do this! I'll update the PR.

Hmmm, good question. I tested with the link you gave me ( https://archive.org/details/2008ELMValidityStudyFinalReportRevised ), and it looks like BR does a full refresh. Right now, I'm certain BookReader would behave oddly if instantiated twice in the same page with different overrides, but this has always been the case. Long term it would be nice if this _wasn't_ the case (would help with testing as well if it was less globally afixed),